ui/k-info-panel/switch_info.kytos

```diff
@@ -17,7 +17,8 @@
       <k-accordion-item title="Interfaces" v-if="this.interfaces">
          <k-interface :interface_id="interface.id" :name="interface.name" :port_number="interface.port_number" :mac="interface.mac" 
                       :speed="interface.speed" :key="interface.name" :enabled="interface.enabled" :metadata="interface.metadata"
-                      :active="interface.active" :lldp="interface.lldp" :nni="interface.nni" :uni="interface.uni" :content_switch="content_switch" v-for="interface in this.interfaces">
+                      :active="interface.active" :lldp="interface.lldp" :nni="interface.nni" :uni="interface.uni" :content_switch="content_switch" 
+                      :available_tags="interface.available_tags" :tag_ranges="interface.tag_ranges" v-for="interface in this.interfaces">
```

Comment:
The new `k-interface` `available_tags` and `tag_ranges` props look great. However, I'll recommend that you instead of setting `interface.available_tags` and `interface.tag_ranges` you fetch directly from `GET /api/kytos/topology/v3/interfaces/tag_ranges?dpid=dpid`, and then you add a support for filtering by dpid on your PR https://github.com/kytos-ng/topology/pull/166/files, and then when this `switch_info.kytos` component gets mounted you fetch it, we'll only have to pay one extra request, but then the positive side is that we do change the existing v3 and v3/interfaces response contents while still not adding too extra data on existing endpoints, so the `interface.as_dict()` change on PR https://github.com/kytos-ng/kytos/pull/421 shouldn't be needed. Wdyt? Let me know if you see another approach other than also changing the response of the existing v3 endpoints. Thanks, @Alopalao. 

Side note: At amLight, in prod, typically each switch will have 32 interfaces, and a dozen of switches, as EVCs start allocating and specially when UNI `available_tags` might get partitioned it might get a bit lengthy (I'm thinking of 100+ EVPLs for instance using non-contiguous values), so I'm a bit concerned to include `tag_ranges` and `available_tags` on `interface.as_dict()`, even though it's well intregrated and makes a lot of sense.

Reply:
Sure, it is a minor change. With that change this PR no longer needed though, so I am closing this.